mission_manager/include/mission_manager/behaviors/altitude_heading.h

```diff
@@ -34,41 +34,41 @@
 
 // Original version: Christopher Scianna Christopher.Scianna@us.QinetiQ.com
 
-#ifndef __ALTITUDE_HEADING_H
-#define __ALTITUDE_HEADING_H
-
-#include "../behavior.h"
+/*      Behavioral - Altitude Heading
+
+        <altitude_heading>
+            <description>
+                00:00:00 - .
+            </description>
+            <when unit="sec">0</when>
+            <timeout unit="sec">50</timeout>
+            <altitude unit="deg">10.0</altitude>
+            <heading unit="deg">180.0</heading>
+            <speed_knots>0.0</speed_knots>
+        </altitude_heading>
+*/
```

Comment:
@patapufate I was thinking of putting it right next to the class definition, as Doxygen block, but OK.